package.json

```diff
@@ -9,35 +9,31 @@
     "npm": ">= 8.19.4"
   },
   "scripts": {
+    "lg": "lg",
```

Comment:
do we need to define this here, or is it a global install on your OS? (not opposed, just curious!)

Reply:
No ,I just had this here for testing